flang/test/Driver/arch-specific-libdir-rpath-by-default.f95

```diff
@@ -0,0 +1,34 @@
+! REQUIRES: x86-registered-target
```

Comment:
This test doesn't actually need x86-registered-target. The driver is able to construct command lines for targets that aren't registered as LLVM backends, and since we're using -### all we need is for the driver to construct the command lines.

Reply:
removed those requirements